varia/inc/color-utils.php

```diff
@@ -0,0 +1,46 @@
+<?php 
```

Comment:
Might make sense to call this file `wpcom-color-utils.php` for consistency. But we never submitted Varia to the themes directory so I could go either way.

Reply:
I didn't go that route because these functions are not wpcom specific and they can be used in other places. In fact they are used inside `functions.php` and even though Varia + children are wpcom only I wanted to keep them consistent. If they were to be released to .org, these functions would be used for the color customizer that core uses instead of the color annotation plugin

Reply:
Got it!